src/main/java/mate/jdbc/dao/impl/DriverDaoImpl.java

```diff
@@ -37,7 +38,7 @@ public Driver create(Driver driver) {
 
     @Override
     public Optional<Driver> get(Long id) {
-        String query = "SELECT id, name, license_number "
+        String query = "SELECT id, name, licenseNumber "
```

Comment:
Why did you change license_namber name to licenseNumber? In SQL it's a bad naming